skyportal/handlers/api/followup_request.py

```diff
@@ -1365,17 +1411,64 @@ async def get(self, instrument_id):
             else:
                 observation_end = Time(arrow.get(observation_end_date.strip()).datetime)
 
-            followup_requests = followup_requests.options(
-                joinedload(FollowupRequest.allocation).joinedload(
-                    Allocation.instrument
-                ),
-                joinedload(FollowupRequest.allocation).joinedload(Allocation.group),
-                joinedload(FollowupRequest.obj),
-                joinedload(FollowupRequest.requester),
-            )
+            if not standards_only:
+                allocation_query = Allocation.select(self.current_user).where(
+                    Allocation.instrument_id == instrument_id
+                )
+                allocation_subquery = allocation_query.subquery()
 
-            followup_requests = session.scalars(followup_requests).unique().all()
-            if len(followup_requests) == 0:
+                # get owned assignments
+                followup_requests = FollowupRequest.select(self.current_user).join(
+                    allocation_subquery,
+                    FollowupRequest.allocation_id == allocation_subquery.c.id,
+                )
+
+                if start_date:
+                    start_date = str(arrow.get(start_date.strip()).datetime)
```

Comment:
Not sure if you need to cast to string, I think `created_at` is a datetime object.

Reply:
ok.